docs/content/routing/routers/index.md

```diff
@@ -50,8 +52,8 @@ In the process, routers may use pieces of [middleware](../../middlewares/overvie
 
 ### EntryPoints
 
-If not specified, HTTP routers will accept requests from all defined entrypoints.
-If you want to limit the router scope to a set of entrypoints, set the entrypoints option.
+If not specified, HTTP routers will accept requests from all defined entry points.
+If you want to limit the router scope to a set of entry points, set the entry points option.
```

Comment:
```suggestion
If you want to limit the router scope to a set of entry points, set the `entryPoints` option.
```

I think the last one is meant to represent the actual option? So you shouldn't split that into two words? Backticks to further emphasize it is an option/property.